coderd/provisionerdserver/provisionerdserver_test.go

```diff
@@ -482,19 +502,32 @@ func TestFailJob(t *testing.T) {
 	})
 	t.Run("WorkspaceBuild", func(t *testing.T) {
 		t.Parallel()
-		srv := setup(t)
-		build, err := srv.Database.InsertWorkspaceBuild(ctx, database.InsertWorkspaceBuildParams{
+		// Ignore log errors because we get:
+		//
+		//	(*Server).FailJob       audit log - get build {"error": "sql: no rows in result set"}
+		ignoreLogErrors := true
+		srv := setup(t, ignoreLogErrors)
```

Comment:
I invite extra scrutiny to this test here! **The audit log implementation may not have ideal behavior.** Please ensure that we actually shouldn't audit log when a workspace build wasn't created.